src/main/scala/com/gyoho/adget/AdgetService.scala

```diff
@@ -0,0 +1,30 @@
+package com.gyoho.adget
+
+import com.gyoho.adget.utils.ScalaHttpClient
+
+object AdgetService {
+  @volatile private var ads: Map[String, Ad] = Map.empty
```

Comment:
class with state => actor model

use message passing to handle the concurrency instead of `@volatile`